src/Components/Articles.jsx

```diff
@@ -6,25 +6,50 @@ import { Link } from "react-router-dom";
 import { fetchArticles } from "./api";
 
 export default function Articles() {
+  const [sortBy, setSortBy] = useState("created_at");
+  const [orderBy, setOrderBy] = useState("desc");
   const [articleList, setArticleList] = useState([]);
   const [page, setPage] = useState(1);
   const [loading, setLoading] = useState(true);
 
   useEffect(() => {
     setLoading(true);
-    fetchArticles(page).then(({ data: { articles } }) => {
+    fetchArticles(page, sortBy, orderBy).then(({ data: { articles } }) => {
       setArticleList(articles);
       window.scrollTo(0, 0);
       setLoading(false);
     });
-  }, [page]);
+  }, [page, sortBy, orderBy]);
 
   return loading ? (
     <h1>Loading...</h1>
   ) : (
     <div>
-      <h1>Articles</h1>
       <ChangePage setPage={setPage} page={page} />
+      <h1>Articles</h1>
+      <div>
+        <div className="sort-bar">
+          <label className="sort-label">Sort By:</label>
```

Comment:
Sort by comment count does not seem to work and you get a 500 error logged in the console 🙂